src/sagemaker/processing.py

```diff
@@ -113,6 +113,7 @@ def __init__(
     def run(
         self,
         inputs=None,
+        kms_key=None,
```

Comment:
maybe later in the arg list. if a customer is relying on ordinal position rather naming args on invocation, then this would be a breaking change.